aspnetcore/blazor/javascript-interoperability/call-dotnet-from-javascript.md

```diff
@@ -2135,6 +2135,14 @@ In the preceding example:
 
 [!INCLUDE[](~/blazor/includes/js-interop/6.0/size-limits.md)]
 
+## .NET JavaScript `[JSImport]`/`[JSExport]` interop
+
+*This section applies to Blazor WebAssembly apps.*
+
+As an alternative to interacting with JavaScript (JS) in Blazor WebAssembly apps using Blazor's JS interop mechanism based on the <xref:Microsoft.JSInterop.IJSRuntime> interface, a .NET JS `[JSImport]`/`[JSExport]` interop API is available to apps targeting ASP.NET Core 7.0 or later.
```

Comment:
ASP.NET Core is really part of .NET, so it's really .NET that carries the version. I prefer language like "ASP.NET Core in .NET 7" to avoid folks thinking that ASP.NET Core versions or releases independently of the rest .NET.